narwhals/_arrow/series.py

```diff
@@ -393,7 +392,8 @@ def is_in(self, other: Any) -> Self:
         return self._from_native_series(pc.is_in(ser, value_set=value_set))
 
     def arg_true(self) -> Self:
-        np = get_numpy()
+        import numpy as np  # ignore-banned-import
```

Comment:
currently, `import pyarrow` also imports NumPy

but I don't know if that's something we can rely on never changing in the future

so I think we should explicitly import numpy here

i'll add some tests regarding imports, that's probably better than relying on these `get_` functions